spec/class.dd

```diff
@@ -729,6 +732,11 @@ $(GNAME Destructor):
         by the gc.
         )
 
+        $(P The compiler may generate $(D __dtor) and $(D __xdtor) alias that
+        matches the class destructor definition and inclusive destructors,
```

Comment:
is the term "inclusive destructor" defined elsewhere in the spec? If not I think it should have a definition somewhere. 

I note that `__xpostblit()`'s description in the struct page of the spec mentions the creation of aliases. 
>void __xpostblit(). The field and aggregated postblits, although generated for a struct, are not actual struct members. In order to be able to call them, the compiler internally creates an alias, called __xpostblit which is a member of the struct and which points to the generated postblit that is the most inclusive.

However, I note that this wording seems to be descriptive rather than prescriptive, which isn't ideal for the language specification.

Reply:
I added the `inclusive destructor` concept to the glossary. Please review it.

About the `__xpostblit()`, I think it is a bit out of scope of this PR, although I can write a separate one fixing that wording.